### ISSUE TITLE
[Misc] Dragonwell feature support on mutiple plantforms

### ISSUE THREAD

**Issue by j0826** [alibaba/dragonwell8#281]
- 不支持: 无法正常启动
- 支持: 保证单元测试, 未生产环境验证
- 生产可用

|  功能\平台   | windows  | linux_x64 | linux_aarch64 | linux_alpine
|  ----  | ----  | ----  | ----  | ----  |
| Wisp | 不支持 | 生产可用 | 生产可用 | 不支持

**Comment by linghengqian** [comment]
I have a question, if the `linux_alpine` platform is unable to use `Alibaba DragonWell`,  mentioned `registry.cn-hangzhou.aliyuncs.com/dragonwell/dragonwell8:8.5.5-FP1_alpine_x86_64_8u275-b2` image in the webpage of https://github.com/alibaba/dragonwell8/wiki/Use-Dragonwell8-docker-image#list-of-dragonwell-images cannot use DragonWell features? ( Take `Wisp2` as an example )

**Comment by joeyleeeeeee97** [comment]
@linghengqian 
Yes, you are right. Please use linux container if you are interested in Dragonwell features.
```
Or you could try https://wiki.alpinelinux.org/wiki/Running_glibc_programs. 
Notice that Dragonwell11 provides musl based binary without any internal features, and current alpine linux Dragonwell 8 image is a glibc based binary assembled to alpine system..
```
